Sources/UberAuth/Authorize/AuthorizationCodeAuthProvider.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
+//
+
+import AuthenticationServices
+import Foundation
+import UberCore
+
+public final class AuthorizationCodeAuthProvider: AuthProviding {
+    
+    // MARK: Initializers
+    
+    public init() {}
+    
+    // MARK: Public Properties
+    
+    public typealias Completion = (Result<Client, UberAuthError>) -> Void
```

Comment:
This is also being defined somewhere else as `AuthCompletion`. Can we avoid the duplication?

Reply:
Good catch. Fixed.